package/scripts/postinstall

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# verify the files exist
+tmp_brew="$2"
+if [[ ! -d "${tmp_brew:?}" ]]; then
+    echo "no directory at $tmp_brew, exiting"
+    exit 1
+fi
+
+# pick the correct target
+if [[ "$3" != "/" ]]; then
+  target=$3
+elif [[ $(uname -m) == "x86_64" ]];then
+  target="/usr/local"
+else
+  target="/opt/Homebrew"
+fi
```

Comment:
What happens if the user wants a Rosetta install? There's a signficant portion of users who do.

Does running the installer as Rosetta work?

Or would it be easier to supply separate -x86_64 and -arm64 pkgs so it's more obvious? Would mean not having to deal with `/tmp`.

Reply:
I'm not sure but it's not a scenario I want to entertain in the first version, just like it isn't entertained in the oneliner in brew.sh

Reply:
> it isn't entertained in the oneliner in brew.sh

Because the oneliner is the same for both arm64 and Rosetta.

From a UX point of view, it has brought many support questions about users being unsure about what version of Homebrew they have.

Reply:
I still consider that an unsupported case for this iteration of the installer. If you want a non-native install you can do a manual one.

Reply:
This constrain seems reasonable to me.